app/views/projects/_project.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class='col-md-12 col-12 col-sm-12'>
+  <div class='card'>
+    <h5 class='card-header'>Created at: <%= formatted_date(@project.created_at) %></h5>
+    <div class='card-body'>
+      <h5 class='card-title'><b>Title: </b> <%= @project.title %></h5>
+      <p class='card-text'><b>Project: </b> <%= @project.description %></p>
+      <p class='card-text'><b>Active: </b> <%= @project.active %></p>
+      <p class='card-text'><b>Status: </b> <%= @project.project_type %></p>
+      <p class='card-text'><b>Start Date: </b> <%= formatted_date(@project.started_at) %></p>
+      <p class='card-text'><b>Close Date: </b> <%= formatted_date(@project.close_at) %></p>
+      <%= link_to 'Edit', [:edit, current_user, @project], class: 'btn btn-success' %>
+      <%= link_to 'Back', user_projects_path, class:'btn btn-primary' %>
+      <%= link_to 'Delete', [current_user, @project], data: { turbo_method: :delete, turbo_confirm: 'Are you sure?'}, class: 'btn btn-danger' %>
```

Comment:
can you see a missing space 
`data: { turbo_method: :delete, turbo_confirm: 'Are you sure?'}`

Reply:
done